pexec/managed_process.go

```diff
@@ -154,10 +154,6 @@ func (p *managedProcess) Start(ctx context.Context) error {
 		return err
 	}
 
-	if _, err := exec.LookPath(p.name); err != nil {
```

Comment:
See this great comment by Dan digging into this: https://github.com/viamrobotics/goutils/commit/6caf7e838efb651fb2d5365803d3ff26f320191f#r144435005